doc/cephadm/services/monitoring.rst

```diff
@@ -184,10 +184,80 @@ To see the default container images, run a command of the following form:
    DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.27.0'   
    DEFAULT_GRAFANA_IMAGE = 'quay.io/ceph/grafana:10.4.0'
 
+``cephadm`` stores a local copy of the ``cephadm`` binary in
+``var/lib/ceph/{FSID}/cephadm.{DIGEST}``, where ``{DIGEST}`` is an alphanumeric
+string representing the currently-running version of Ceph. In the Squid and
+Reef releases of Ceph, this ``cephadm`` file is stored as a zip file and must
+be unzipped before its contents can be examined.
+
+This procedure explains how to generate a list of the default container images
+used by ``cephadm``.
+
+.. note:: This procedure applies only to the Reef and Squid releases of Ceph.
+   If you are using a different version of Ceph, you cannot use this procedure
+   to examine the list of default containers used by cephadm. Make sure that
+   you are reading the documentation for the release of Ceph that you have
+   installed.
+
+#. To create a directory called ``cephadm_dir``, run the following command:
+
+   .. prompt:: bash #
+
+      mkdir cephadm_dir
+
+#. To unzip ``/var/lib/ceph/{FSID}/cephadm.{DIGEST}`` in the directory
+   ``cephadm_dir``, run a command of the following form:
+
+   .. prompt:: bash #
+
+      unzip /var/lib/ceph/{FSID}/cephadm.{DIGEST} -d cephadm_dir > /dev/null
+
+   ::
+
+      warning [/var/lib/ceph/{FSID}/cephadm.{DIGEST}]:  14 extra bytes at
+      beginning or within zipfile
+      (attempting to process anyway)
+
+
+#. To use ``egrep`` to search for the string ``_IMAGE`` in the file
+   ``cephadm_dir_cephadmlib/constants.py``, run the following command: 
+
+   .. prompt:: bash #
+
+      egrep "_IMAGE" cephadm_dir/cephadmlib/constants.py
+
+   ::
+
+      DEFAULT_IMAGE = 'quay.ceph.io/ceph-ci/ceph:main'
```

Comment:
Just a note, this is accurate to what would get outputted currently, but I'll need to adjust this to point to `quay.ceph.io/ceph-ci/ceph:squid` at some point. I'll make sure to adjust this line in the docs when that happens.